src/client/components/ui/Select/ListBox.tsx

```diff
@@ -68,6 +70,9 @@ export function ListBox(props: ListBoxProps) {
               label={option.libellé}
               value={option.valeur}
               onChange={() => {
+                if(closeOnExit){
+                  setIsOptionOpen(false);

```

Comment:
```suggestion
                  setIsOptionOpen && setIsOptionOpen(false);

```

`setIsOptionOpen` peut être undefined, du coup il faut vérifier qu'il existe avant de l'appeler !

Reply:
en fait je pense qu'il faut virer complètement `closeOnExit` et `setIsOptionOpen` de ce composant =). 3 lignes plus bas on a 
```typescript
if (onChange)  {
  onChange(option.valeur)
}
```
qui appelle déjà le composant parent (ici notre Select) qui pourra prendre la décision de fermer les options dans sa fonction `onChange`.